hooks/test_post_gen_project.py

```diff
@@ -49,3 +56,57 @@ def test_copy_journal_template_files_bad(
         match=errormessage,
     ):
         copy_journal_template_files(input, project_dir)
+
+
+# C1: There are bib files in the cloned directory.
+#   Expect all bib name are inserted into manuscript,
+#   and bib files are copied into manuscript's parent dir
+#   exists.
+# C2: There are no bib files in the cloned directory.
+#   Expect manuscript bib entries will not be modified,
+#   and no files are copied.
+def test_load_bib_info(user_filesystem, template_files):
+    source_dir = user_filesystem / "user-repo-dir"
+    manuscript_path = (
+        user_filesystem
+        / ".cookiecutters"
+        / "scikit-package-manuscript"
+        / "templates"
+        / "article"
+        / "manuscript-in-spm.tex"
+    )
+    load_bib_info(source_dir, manuscript_path)
+    actual_manuscript_content = manuscript_path.read_text()
+    expected_manuscript_content = r"""
+\documentclass{article}
+\usepackage{package-in-manuscript}
+\newcommand{\command_in_manuscript}[1]{\mathbf{#1}}
+\begin{document}
+Contents of manuscript
+\bibliography{user-bib-file-1, user-bib-file-2}
+\bibliography{bib-in-manuscript}
+\bibliographystyle{chicago}
+\end{document}
+    """
+    assert expected_manuscript_content == actual_manuscript_content
+    manuscript_path.write_text(template_files["manuscript-in-spm.tex"])
+
+    Path(source_dir / "user-bib-file-1.bib").unlink()
+    Path(source_dir / "user-bib-file-2.bib").unlink()
+    load_bib_info(source_dir, manuscript_path)
+    actual_manuscript_content = manuscript_path.read_text()
+    expected_manuscript_content = template_files["manuscript-in-spm.tex"]
+    assert expected_manuscript_content == actual_manuscript_content
+    manuscript_path.write_text(template_files["manuscript-in-spm.tex"])
+
+
+# C1: The manuscript path doesn't exist. Expect file not found error.
```

Comment:
I don't think this one is needed.  Especially if this is run after the `load_headers` which will crash for hte same thing.